docs/03-04-function-configuration-file.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Function's configuration file
```

Comment:
Personally, I think `Function configuration file` sounds better 🤔 But this is also correct, so feel free to leave it.